charter.md

```diff
@@ -0,0 +1,45 @@
+There are many situations in which it is desirable to take
+measurements of data which people consider sensitive. For instance,
+one might want to measure the most common sites that people browse to
+or exposure to some disease. In these cases, the entity taking the
+measurement is not interested in people's individual responses but
+rather in aggregated data (e.g., how many users visit URL X), but
+conventional methods require collecting individual responses and then
+aggregating them, thus representing a threat to user privacy and
+rendering many such measurements difficult and impractical.
+
+New cryptographic techniques such as Prio and Heavy Hitters address
+this gap by splitting up measurements between multiple servers which
+can jointly compute the aggregate value without either server learning
+the value of individual responses. The Private Data Aggregation (PDA)
+work will standardize a protocol for deployment of these techniques on
+the Internet. This protocol will include mechanisms for:
+
+- Client submission of individual reports, including proofs of validity.
+- Server verification of validity proofs
+- Server computation of aggregate values and collection of results to
+- Reporting of aggregate results to the entity taking the measurement
+  
+Configuration of clients and servers is out of scope for the working
+group. It is assumed that this happens out of band as part of the
+PDA service. 
+
+The WG will deliver a protocol which can accommodate multiple PDA
```

Comment:
a protocol?  seems like something that a working group might decide, but HH and Prio are very different things functionally.  my take: make two, share as many design elements as possible